packages/next-codemod/lib/codemods.ts

```diff
@@ -0,0 +1,96 @@
+type Codemod = {
+  title: string
+  value: string
+}
+
+type VersionCodemods = {
+  version: string
+  codemods: Codemod[]
+}
+
+export const availableCodemods: VersionCodemods[] = [
```

Comment:
Missing v15, could leverage the existing mapping in codemod bin to avoid declare them again